lib/src/specs/expression/literal.dart

```diff
@@ -8,30 +8,15 @@ part of '../expression.dart';
 ///
```

Comment:
I think its probably worth adding a comment that Expression is also handled and why

Reply:
Done.